plugins/woocommerce/client/admin/client/settings-payments/components/payment-settings-section/payment-settings-section.scss

```diff
@@ -0,0 +1,56 @@
+.payment-settings-section {
+  display: flex;
+  flex-flow: row;
+
+  &:not( &:last-child ) {
+    margin-bottom: 24px;
+  }
+
+  @include breakpoint( '>800px' ) {
```

Comment:
I believe this will be ignored and do nothing. breakpoint mixin only supports limited breakpoints.

https://github.com/woocommerce/woocommerce/blob/5471680004b7c832c9e47b7514ddd229852ad8fd/packages/js/internal-style-build/abstracts/_breakpoints.scss#L10

If you need to use this specific breakpoint, you can use media query directly.

Reply:
Thanks, great catch. I updated to use 782 instead as it's probably what I should have used initially.